From 242d651004f806dacd27c6690b6aed3e21e91896 Mon Sep 17 00:00:00 2001
From: Freddie Jheng <Freddie.Jheng@quantatw.com>
Date: Wed, 22 Jul 2026 18:32:43 +0800
Subject: [PATCH] configurations: santabarbara: add MCTPI2CTarget for SWB PEX
 on vr-sni

Add the MCTPI2CTarget configuration to the vr-sni PCIe switch board
variant to allow mctpreactor to manage EID assignment for SWB_PEX
automatically.

Ref: https://gerrit.openbmc.org/c/openbmc/entity-manager/+/92609
Change-Id: I553e6d96980f3642285f227395fd64fd3b251fc8
Signed-off-by: Freddie Jheng <Freddie.Jheng@quantatw.com>
---
 .../santabarbara_pcie_switch_board_brcm_vr_sni.json         | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/configurations/meta/santabarbara/santabarbara_pcie_switch_board_brcm_vr_sni.json b/configurations/meta/santabarbara/santabarbara_pcie_switch_board_brcm_vr_sni.json
index 54c0043..0f06a64 100644
--- a/configurations/meta/santabarbara/santabarbara_pcie_switch_board_brcm_vr_sni.json
+++ b/configurations/meta/santabarbara/santabarbara_pcie_switch_board_brcm_vr_sni.json
@@ -425,6 +425,12 @@
             "Name": "Santabarbara_SWB_CPLD",
             "Type": "LatticeLCMXO3D_9400Firmware"
         },
+        {
+            "Address": "0x62",
+            "Bus": "114",
+            "Name": "SWB_PEX",
+            "Type": "MCTPI2CTarget"
+        },
         {
             "CheckHysteresisWithSetpoint": true,
             "Class": "temp",
-- 
2.53.0-Meta

